url.bs

```diff
@@ -635,6 +635,10 @@ concepts.
  <li><p>Return <var>result</var>.
 </ol>
 
+<p class=note>This document and the web platform at large use
+<cite>Unicode IDNA Compatibility Processing</cite> and not <cite>IDNA2008</cite>. For instance,
```

Comment:
It looks like IDNA2008 is not a document name, but instead an informal term for a collection of documents? The relevant document name is instead "[Internationalized Domain Names for Applications (IDNA): Definitions and Document Framework](https://www.rfc-editor.org/rfc/rfc5890)".

Reply:
So I want to mention IDNA2008. I guess I could just not use `<cite>`. Would I have to quote it?

Reply:
It seems fine to just not use `<cite>`: `and not IDNA2008`.